src/Apps/Legion/Components/TypeIcon/TypeIcon.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { ITypeIconStyles } from './TypeIcon.types';
+import {
+    IProcessedStyleSet,
+    mergeStyleSets,
+    memoizeFunction
+} from '@fluentui/react';
+
+export const getStyles = memoizeFunction(
+    (color: string): IProcessedStyleSet<ITypeIconStyles> => {
+        return mergeStyleSets({
+            root: {
+                display: 'inline-flex',
+                width: 32,
+                height: 32,
+                borderRadius: 16,
```

Comment:
tip: you can use 50% here and it will stay a circle no matter what width and height you give it.